lib/rom/rails/model/form.rb

```diff
@@ -36,11 +37,17 @@ def save(*args)
       end
 
       def success?
-        !errors.any?
+        errors.nil? || !errors.any?
+      end
+
+      def validate!
+        validator = self.class::Validator.new(params)
```

Comment:
it should receive processed attributes rather than raw params, so maybe something like:

``` ruby
def validate!
  validator = self.class.validator.new(attributes)
  validator.validate
  @errors = validator.errors
end

def attributes
  self.class.params[params]
end
```